packages/button/src/ClearButton.ts

```diff
@@ -64,11 +64,53 @@ export class ClearButton extends SizedMixin(StyledButton, {
         return [...super.styles, buttonStyles, crossMediumStyles];
     }
 
+    @property({ type: Boolean, reflect: true })
+    public quiet = false;
+
     /**
      * The visual variant to apply to this button.
+     * @deprecated Use `static-color='white'` instead.
      */
     @property({ reflect: true })
-    public variant: 'overBackground' | '' = '';
+    public set variant(variant: 'overBackground' | undefined) {
```

Comment:
@rubencarvalho @nikkimk If you have a chance, do you know if there's a more efficient or "lit-ified" way to handle this backwards compatibility?